File: Programar em Java/17-CicloForeach.java
// YouTube: https://youtu.be/X8hVHwqEiKI
// Publicação: https://caffeinealgorithm.com/blog/20220228/ciclo-foreach-em-java/

package com.caffeinealgorithm.programaremjava;

import java.util.ArrayList;
import java.util.List;

public class CicloForeach {
  public void Run() {
    List<String> paises = new ArrayList<>();

    paises.add("Portugal");
    paises.add("Brasil");
    paises.add("Espanha");
    paises.add("França");
    paises.add("Itália");
    paises.add("Austrália");
    paises.add("Índia");

    for (String pais : paises)
      System.out.printf("País: %s\n", pais);
  }
}

/*
  País: Portugal
  País: Brasil
  País: Espanha
  País: França
  País: Itália
  País: Austrália
  País: Índia
*/

File: Programar em C#/17-CicloForeach.cs
// YouTube: https://youtu.be/seMBMqxe8KA
// Publicação: https://caffeinealgorithm.com/blog/ciclo-foreach-em-csharp/

using System;
using System.Collections.Generic;

namespace Base {
  class CicloForeach {
    public void Run() {
      string organizacao = "Caffeine Algorithm";
      List<string> paises = new List<string>() {
        "Portugal",
        "Brasil",
        "Espanha",
        "França",
        "Itália",
        "Austrália",
        "Índia"
      };

      /*
        foreach (char carater in organizacao) {
          Console.WriteLine($"Carater: {carater}");
        }
      */

      foreach (string pais in paises) {
        Console.WriteLine($"País: {pais}");
      }
    }
  }
}

/*
  País: Portugal
  País: Brasil
  País: Espanha
  País: França
  País: Itália
  País: Austrália
  País: Índia
*/
